### DISCUSSION TITLE
Error during XML parsing: not well-formed

### DISCUSSION THREAD

**Discussion by shangliang** [MadAnalysis/madanalysis5#217] [Q&A]
When I use the analysis of atlas_susy_2018_32 in the version v 1.10.10, I get the following error,

MA5-WARNING: Error during XML parsing: not well-formed (invalid token): line 1, column 0
MA5-WARNING: Cannot parse the info file
MA5-WARNING: Info file for atlas_susy_2018_32 missing or corrupted. Skipping the CLs calculation.
MA5-ERROR: job submission aborted.

I download the atlas_susy_2018_32_all.json file and put it into the PAD file, the error still exists.

**Comment by shangliang** [comment]
Hi, the file atlas_susy_2018_32.info is not correct XML format.
Could you please check it?

**Comment by shangliang** [comment]
[atlas_susy_2018_32.pdf](https://github.com/MadAnalysis/madanalysis5/files/12524116/atlas_susy_2018_32.pdf)

**Reply by shangliang** [reply]
This is the correct format

**Comment by jackaraz** [comment]
Hi @shangliang, only today, I received too many emails just from the same thread you are constantly messaging to. Our job is not to respond to your messages; we have other things to do. You will receive an answer when we have time and not before that. So please form your question properly **ONCE** and wait for us to have time to respond. This platform is not a Facebook chat box, so please keep it professional.

**Comment by shangliang** [comment]
Dear Jack,I apologize sincerely for the inconvenience caused. I am new to recasting mode of MA5 and also the discussing block at github. I am truly sorry and I want to assure you that this won’t happen again.Best regards,Liang发自我的iPhone------------------ Original ------------------From: Jack Y. Araz  ***@***.***>Date: Tue,Sep 5,2023 3:33 PMTo: MadAnalysis/madanalysis5 ***@***.***>Cc: shangliang ***@***.***>, Mention ***@***.***>Subject: Re: [MadAnalysis/madanalysis5] Error during XML parsing: notwell-formed (Discussion #217)
Hi @shangliang, only today, I received too many emails just from the same thread you are constantly messaging to. Our job is not to respond to your messages; we have other things to do. You will receive an answer when we have time and not before that. So please form your question properly ONCE and wait for us to have time to respond. This platform is not a Facebook chat box, so please keep it professional.

—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you were mentioned.Message ID: ***@***.***>
[
{
***@***.***": "http://schema.org",
***@***.***": "EmailMessage",
"potentialAction": {
***@***.***": "ViewAction",
"target": "https://github.com/orgs/MadAnalysis/discussions/217#discussioncomment-6914542",
"url": "https://github.com/orgs/MadAnalysis/discussions/217#discussioncomment-6914542",
"name": "View Discussion"
},
"description": "View this Discussion on GitHub",
"publisher": {
***@***.***": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
